### PR TITLE
Fix saving changes on #touch

### PR DESCRIPTION
**Pull request by fukayatsu**
from: https://github.com/paper-trail-gem/paper_trail/issues/1073#issuecomment-388766519

Related to #1076

### CODE REVIEW

File: spec/models/on/touch_spec.rb [hunk 1]
```diff
@@ -20,6 +20,12 @@ module On
         )
         expect(record.versions.last.event).to eq("update")
       end
+
+      it "saves a object" do
+        record = described_class.create(name: "Alice")
+        record.touch
+        expect(record.versions.last.reify.name).to eq("Alice")
+      end
```
Comment (fukayatsu) [inline spec/models/on/touch_spec.rb]: On master branch, this spec passes on AR4.2 and AR5.0 but fails on AR5.1 and AR 5.2

Please see https://travis-ci.org/fukayatsu/paper_trail/builds/378643661 .

**Comment by jaredbeck** [comment]
Thanks for the PR, Fukaya-san. The test you added is good.

My only concern is the `in_after_callback:` argument. 

```diff
-        r.paper_trail.record_update(force: true, in_after_callback: true)
+        r.paper_trail.record_update(force: true, in_after_callback: false)
```

PT installs an `after_touch` callback. It is, *actually* an after callback. So, if we say `in_after_callback: false`, that would be confusing. 

First, I thought maybe we should change the name of the argument? But, it's not easy to come up with a better name.

How about just passing another argument?

```diff
-        r.paper_trail.record_update(force: true, in_after_callback: true)
+        r.paper_trail.record_update(force: true, in_after_callback: true, is_touch: true)
```

Please see https://github.com/paper-trail-gem/paper_trail/pull/1085

**Comment by fukayatsu** [comment]
@jaredbeck Thank you for the quick response. 
#1085 looks good to me!

**Comment by jaredbeck** [comment]
Fixed by #1085, released in 9.0.2. Thanks, Fukaya-san.

**Pull request closed without merging**